test/unit/testConfig.js

```diff
@@ -165,9 +167,9 @@ config.ngsi = {
             }
         }
     },
-    service: 'smartGondor',
+    service: 'smartgondor',
     subservice: '/gardens',
-    providerUrl: 'http://192.168.56.1:4041/NGSI10',
+    providerUrl: 'http://localhost:4041/NGSI10',
```

Comment:
Does the test work with /v1 instead of /NGSI10? Each time /NGSI10 is used, a kitten dies ;)

Reply:
Fixed in 0de8f251d1c82832cb5ee4b3104fd86100517c3d